demos/multi_channel/CMakeLists.txt

```diff
@@ -23,3 +23,4 @@ endif()
 add_subdirectory(common)
 add_subdirectory(face_detection_demo)
 add_subdirectory(human_pose_estimation_demo)
+add_subdirectory(yolo_v3)
```

Comment:
I think for consistency with other demos this should be called `object_detection_demo_yolov3` (with the name of the executable updated accordingly).